mlflow/tracking/client.py

```diff
@@ -331,18 +896,104 @@ def set_terminated(self, run_id, status=None, end_time=None):
 
         :param status: A string value of :py:class:`mlflow.entities.RunStatus`.
                        Defaults to "FINISHED".
-        :param end_time: If not provided, defaults to the current time."""
+        :param end_time: If not provided, defaults to the current time.
+
+        .. code-block:: python
+            :caption: Example
+
+            from mlflow.tracking import MlflowClient
+
+            def print_run_info(r):
+                print("run_id: {}".format(r.info.run_id))
+                print("status: {}".format(r.info.status))
+
+            # Create a run under the default experiment id "0".
+            # Since this is low-level CRUD operation, this method will create a run.
+            # To end the run, you'll have to explicitly terminate it.
+            client = MlflowClient()
+            experiment_id = "0"
+            run = client.create_run(experiment_id)
+            print_run_info(run)
+            print("--")
+
+            # Terminate the run and fetch updated status. By default,
+            # the status is set to "FINISHED". Other values you can
+            # set are "KILLED", "FAILED", "RUNNING", or "SCHEDULED".
+            client.set_terminated(run.info.run_id, status="KILLED")
+            run = client.get_run(run.info.run_id)
+            print_run_info(run)
+
+        .. code-block:: text
+            :caption: Output
+
+            run_id: 575fb62af83f469e84806aee24945973
+            status: RUNNING
+            --
+            run_id: 575fb62af83f469e84806aee24945973
+            status: KILLED
+        """
         self._tracking_client.set_terminated(run_id, status, end_time)
 
     def delete_run(self, run_id):
-        """
-        Deletes a run with the given ID.
+        """Deletes a run with the given ID.
+
+        :param run_id: The unique run id to delete.
+
+        .. code-block:: python
+            :caption: Example
+
+            from mlflow.tracking import MlflowClient
+
+            # Create a run under an experiment ID "0", which is
+            # the "Default" name of the experiment
```

Comment:
This comment sounds like the default experiment is named "0".

Reply:
I must have missed this one.